Source package : python-pydocstyle
File name       : pydocstyle-6.3.0.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/python-pydocstyle/pydocstyle-6.3.0.tar.gz/sha512/f8473b19ab6ef0b61787875558f9dd6f9f7f1954e1baa0010942af6d4de8dbca30c8c08be6acbf24aadd1c0a601ba9467b747026a6cd22379f0c4b84a38b57c7/pydocstyle-6.3.0.tar.gz
Source URL      : https://files.pythonhosted.org/packages/source/p/pydocstyle/pydocstyle-6.3.0.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-pydocstyle-6.3.0.tar.gz: gzip compressed data, was "pydocstyle-6.3.0.tar", max compression, original size modulo 2^32 174080
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-pydocstyle-6.3.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 399360


Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: docs
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: .github
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: .gitignore
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: .gitpod.yml
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: Makefile
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: MANIFEST.in
Only in source-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: PKG-INFO
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: poetry.lock
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: .pre-commit-hooks.yaml
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: PULL_REQUEST_TEMPLATE.md
diff -r source-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0/pyproject.toml lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0/pyproject.toml
3c3
< version = "6.3.0"
---
> version = "0.0.0-dev"
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: requirements
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: requirements.txt
Only in source-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: setup.py
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0/src: __init__.py
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0/src: tests
Only in lookaside-pydocstyle-6.3.0.tar.gz-extracted/pydocstyle-6.3.0: tox.ini
